lib/checkPullRequestLabels.js

```diff
@@ -14,6 +15,93 @@ var matchChangelogLabelWithRegex = function(changelogLabel) {
   return regExpForChangelogLabel.test(changelogLabel);
 };
 
+/**
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ * @param {String} changelogLabel
+ */
+const canAssignProjectOwner = (pullRequest, changelogLabel) => {
```

Comment:
Add docstring for what the function checks.

Reply:
Done.